master/internal/api_experiment.go

```diff
@@ -1604,6 +1604,161 @@ func (a *apiServer) ContinueExperiment(
 	}, nil
 }
 
+func (a *apiServer) ContinueHPSearchExperiment(
```

Comment:
can we instead try just modifying ```ContinueExperiment``` endpoint to handle the hp search case

I think two CLI commands still make sense here

Reply:
done